shm/inc/Cargo.hpp

```diff
@@ -4,15 +4,22 @@
 
 class Cargo {
 public:
-    Cargo(const std::string&, size_t, size_t);
+    Cargo(const std::string& name, size_t amount, size_t basePrice_) 
```

Comment:
Finally variable names! Good!

Reply:
> Finally variable names! Good!

If we wanna add varaiable names we should change this in all project